python/src/ExponentialFactory_doc.i.in

```diff
@@ -4,16 +4,30 @@
 Available constructor:
     ExponentialFactory()
 
-The parameters are estimated by method of moments:
+Let :math:`\bar{x}` be the minimum mean. 
+Let :math:`x_{(1)}` be the minimum sample value.
+The parameter :math:`\hat{\gamma}` is estimated by method of maximum 
+likelihood, with asymptotic correction for bias:
 
 .. math::
     :nowrap:
 
     \begin{eqnarray*}
-      \displaystyle \Hat{\gamma}_n = x_{(1,n)} - \frac{|x_{(1,n)}|}{2+n}\\
-      \displaystyle \Hat{\lambda}_n= \frac{1}{\bar{x}_n-\Hat{\gamma}_n}
+      \hat{\gamma} = x_{(1)} - \frac{|x_{(1)}|}{2 + n}
     \end{eqnarray*}
 
+The parameters :math:`\hat{\lambda}` is estimated by method of moments:
```

Comment:
Same. Also, it should be "the parameter" (singular).

Reply:
Fixed.